--- clutter-gtk-0.10.8/clutter-gtk/Makefile.am~	2010-09-16 15:44:00.000000000 +0200
+++ clutter-gtk-0.10.8/clutter-gtk/Makefile.am	2010-09-25 15:59:10.178393824 +0200
@@ -47,7 +47,8 @@
 GtkClutter-@CLUTTER_GTK_API_VERSION@.gir: $(INTROSPECTION_SCANNER) Makefile libclutter-gtk-@CLUTTER_GTK_API_VERSION@.la
 	$(AM_V_GEN)$(INTROSPECTION_SCANNER) -v \
 		--namespace GtkClutter --nsversion=@CLUTTER_GTK_API_VERSION@ \
-		--strip-prefix=GtkClutter \
+		--identifier-prefix=GtkClutter \
+		--symbol-prefix=gtk_clutter \
 		$(INCLUDES) $(AM_CPPFLAGS) \
 		--add-include-path=$(srcdir) --add-include=path=. \
 		--include=Clutter-1.0 \
